cpp/include/rapidsmpf/buffer/packed_data.hpp

```diff
@@ -22,39 +24,39 @@ namespace rapidsmpf {
  */
 struct PackedData {
     std::unique_ptr<std::vector<std::uint8_t>> metadata;  ///< The metadata
-    std::unique_ptr<rmm::device_buffer> gpu_data;  ///< The gpu data
+    std::unique_ptr<Buffer> gpu_data;  ///< The gpu data
```

Comment:
*nit*: should we just call this `data` now? It might never have been on gpu.

Reply:
I still like to use `gpu_data`. It might not be located on device, but it is still gpu data?

Reply:
Good point, done